hs_core/page_processors.py

```diff
@@ -375,7 +377,9 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
                'discoverable': discoverable,
                'relation_source_types': tuple((type_value, type_display)
                                               for type_value, type_display in Relation.SOURCE_TYPES
-                                              if type_value != 'isReplacedBy' and type_value != 'isVersionOf')
+                                              if type_value != 'isReplacedBy' and type_value != 'isVersionOf'),
+               'res_id': content_model.short_id,
+               'extra_matadata_dict': content_model.extra_metadata
```

Comment:
@pkdash Same here, I believe you can directly use cm.short_id and cm.extra_metadata in template without passing them in page context.